recipes/icon_font_to_png/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "icon_font_to_png" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 725278e7550183c679fa5bfd1f85566ff3d71205765a57238ea1289e40522a37
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pillow >=4.0.0
+    - pip
+    - python
```

If (and only if!) this can be `noarch: python`:

```suggestion
    - python >=3.6
```